torch/_export/serde/serialize.py

```diff
@@ -242,30 +242,27 @@ def deserialize_signature(sig: GraphSignature) -> ep.ExportGraphSignature:
     )
 
 
-def serialize_state_dict(state_dict: Dict[str, Any]) -> bytes:
+def serialize_torch_artifact(artifact: Union[Dict[str, Any], Iterable[Any]]) -> bytes:
```

Comment:
Is there any danger that original args can be more types of objects than state dict, and thus harder to `torch.save`? Or do you think we have enough restrictions that this is safe and general?

Reply:
I think...torch.save should be able to handle most things? If not then users can not store these args. Otherwise I think it will be too much work on our side to figure out how to serialize different arguments.